.travis.yml

```diff
@@ -1,34 +1,37 @@
-sudo: required
-dist: trusty
+os: linux
+dist: bionic
+
+addons:
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+
+services:
+  - mysql
+  - postgresql
 
 language: ruby
-cache: bundler
 
-addons:
-  postgresql: 9.4
+rvm:
+  - 2.5.3
+  - 2.4.5
+  - 2.3.8
 
 env:
-  - DB=postgres
   - DB=mysql
+  - DB=postgres
 
 gemfile:
   - gemfiles/spree_3_1.gemfile
   - gemfiles/spree_3_7.gemfile
-    #  - gemfiles/spree_3_2.gemfile
-    #  - gemfiles/spree_3_5.gemfile
+#  - gemfiles/spree_3_2.gemfile
+#  - gemfiles/spree_3_5.gemfile
   - gemfiles/spree_4_0.gemfile
```

Comment:
@przemosk please add 4.1 and 4.2 as well

Reply:
@damianlegawiec sure, should I remove those commented versions ?

Reply:
@przemosk yes, let's remove the commented out

Reply:
@damianlegawiec fixed